src/infrastructure/config.ts

```diff
@@ -101,6 +110,21 @@ export function getConfig(): Config {
         isEnabled: configScope.getOptionalBoolean('BUGSNAG_ENABLED', true),
         apiKey: configScope.getOptionalNullable('BUGSNAG_KEY', undefined),
       },
+      amplitude: {
+        isEnabled: configScope.getOptionalBoolean('AMPLITUDE_ENABLED', false),
+        apiKey: configScope.getOptionalNullable('AMPLITUDE_KEY', undefined),
+        serverZone: configScope.getOptionalValidated(
```

Comment:
getOptionalOneOf seems like a more natural fit, but it's not supported yet. Can you add this option to https://github.com/lokalise/node-core/blob/main/src/config/ConfigScope.ts?

Reply:
Nice suggestion! https://github.com/lokalise/node-core/pull/60